### PR TITLE
Repo cleanup (unused data sources, readme updates, etc.)

### PR DESCRIPTION
**Pull request by mikelehen**
I took a fairly heavy-handed approach at cleaning up the repo, deleting lots of data sources we no longer (or never used) and dropping a bunch of stale info from the readme.

Part of me feels bad for deleting all of this, but I really doubt we'll ever need it again and it'll live on in git history.

### CODE REVIEW

**Review by BrettBoval — APPROVED** [review]
I'm fine with removing all of these. A couple small notes about adding documentation to good federal datasources. And actually building a new env with the updated requirements.txt to confirm that it works.

File: requirements.txt [hunk 1]
```diff
@@ -5,22 +5,12 @@ idna==2.9
 pandas==1.0.3
```
[the remark this answers is not in the document]
Comment (mikelehen) [reply requirements.txt:5]: Yep!  I did it both locally and just kicked off the github action on this branch. Both worked.

**Review by smcclure17 — APPROVED** [review]
this all looks good to me 🚀 

just out of curiosity, what were the shapefiles originally used for in this repo?

**Comment by mikelehen** [comment]
Very early on we had a version of the API that used shapefiles so you could import it directly into various tools and get a map showing case numbers / risk levels across the US.  TBH I don't know / remember the full details, but we deprecated that API a long time ago. 😁

**Pull request merged**